### PR TITLE
Add Parsing and Pushing to ES

### PR DESCRIPTION
**Pull request by lamarwhitej**
Added further parsing for persistent test pre and post validation.  Added push to ES.

### CODE REVIEW

File: jenkins/osa_onmetal.functions.groovy [hunk 1]
```diff
@@ -370,6 +370,18 @@ def stop_api_uptime_tests() {
     """
 }
 
+def parse_results() {
+
+    String host_ip = get_onmetal_ip()
+
+    //Pull persistent, during, api, smoke results from onmetal to ES vm
+    sh """
+    git clone https://github.com/arithx/elastic-benchmark
```
Comment (castulo) [inline jenkins/osa_onmetal.functions.groovy]: Why are we cloning from a personal repo? I thought the idea was to either include the files into this project directly (qa-jenkins-onmetal) or to move them to the OSIC repository. We could ask Daryl what he thinks?

**Review by castulo — COMMENTED** [review]
My understanding was that we were going to use only OSIC repo (not personal) for the work we were going to submit. We may need to clarify this with Daryl.

File: jenkins/osa_onmetal.functions.groovy [hunk 2]
```diff
@@ -370,6 +370,18 @@ def stop_api_uptime_tests() {
...
 
+def parse_results() {
+
+    String host_ip = get_onmetal_ip()
+
+    //Pull persistent, during, api, smoke results from onmetal to ES vm
+    sh """
+    git clone https://github.com/osic/elastic-benchmark
+    sudo pip install -e elastic-benchmark
+    elastic-upgrade -u output/output.txt -d output/during_output.txt -p output/persistent_resource.txt -b subunit/before_upgrade -a subunit/after_upgrade
```
Comment (castulo) [inline jenkins/osa_onmetal.functions.groovy:381]: why are you using output/persistent_resource.txt? These are the directories that get created by the smoke tests and persistent resources tests:
<noformat>
- subunit
  - persistent_resources
    - create
    - before_upgrade
    - after_upgrade
    - cleanup
  - smoke
    - before_upgrade
    - after_upgrade 
      <noformat>

Comment (lamarwhitej) [reply jenkins/osa_onmetal.functions.groovy:381]: That is where Anthony's script parses and places the formatted persistence test output.

Comment (castulo) [reply jenkins/osa_onmetal.functions.groovy:381]: got it

**Pull request merged**